src/locales/en.json

```diff
@@ -943,5 +943,6 @@
 "yourInstance": "Your instance",
 "newInstance": "New instance",
 "cpu": "CPU",
-"ram": "RAM"
+"ram": "RAM",
```

Comment:
Can you maybe check against the old UI, wrt any translations you need in the VM deployment form?